features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -410,6 +410,8 @@
   "event.default_listener.user_claim_audit_logger.priority": "9",
   "event.default_listener.user_claim_audit_logger.enable": false,
   "event.default_listener.user_claim_audit_logger.LogUpdatedClaimsOnly": false,
+  "event.default_listener.client_certificate_authentication_handler.priority": "10",
+  "event.default_listener.client_certificate_authentication_handler.enable": true,
```

Comment:
Should we have this enabled by default?

Reply:
By following code, whatever concerete event listenr defined in identity.xml.j2 and toml configured custom event listerenr will be enabled.
https://github.com/wso2/carbon-identity-framework/blob/1e1a732ffad969cca6f90cd01d7b92b927caa1ad/components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityConfigParser.java#L272

Since this event listener is considered as custom event listener, This should be set to false by default